portal/src/main/webapp/js/src/data_download.js

```diff
@@ -200,6 +210,70 @@ var DataDownloadTab = (function() {
         $("#text_area_case_affected").append(strs.case_affected);
         $("#text_area_case_matrix").append(strs.case_matrix);
     }
+    //TODO : include study name(id) incase of sample id match across studies?
```

Comment:
For the text field, could you add a column (first column) to include the study ID?

For the data matrix, add a row or column (transposed). For example, here is the trasposed version:

```
STUDY_ID	SAMPLE_ID	EGFR	TP53
BRCA_TCGA	TCGA-3X-AAV9-01	0	1
BRCA_TCGA	TCGA-3X-AAVA-01	0	-1
BRCA_XXXX	CHOL12		NA	2
```